cocos/gfx/base/pipeline-state.editor.ts

```diff
@@ -319,7 +319,7 @@ export class PassStatesEditor implements EffectAsset.IPassStates {
     @type(BlendStateEditor)
     @serializable
     @editable
-    public blendState: BlendStateEditor = new BlendStateEditor();
+    public blendState: any = new BlendStateEditor();
```

Comment:
I think he just want to avoid this error: 
```
Property 'blendState' in type 'PassStatesEditor' is not assignable to the same property in base type 'IPassStates'.
  Type 'BlendStateEditor' is missing the following properties from type 'BlendState': _token, native, setTarget, reset, destroyts(2416)
(property) PassStatesEditor.blendState: BlendStateEditor
```